packages/brand/Kaizen/Brand/Brand.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { assetUrl } from "../../../hosted-assets"
+
+interface BrandProps {
+  variant:
+    | "logo-horizontal"
+    | "logo-vertical"
+    | "enso"
+    | "collective-intelligence"
+  isReversed: boolean
+}
+
+export const Brand = (props: BrandProps) => {
+  const brandTheme = props.isReversed ? "-reversed" : "-default"
+
+  return (
+    <img
+      src={assetUrl(`brand/${props.variant}${brandTheme}.svg`)}
+      alt="Culture Amp"
```

Comment:
Also nitpicking, and @dougmacknz might be able to advise better, but are we _always_ going to want this `alt` to be `Culture Amp`? I'm a little unsure of where the collective intelligence logo is going to be used.

Reply:
🤔 In general this is what we'll want, except for the collective intelligence one which will need something different. I see a couple of options with pros and cons:
* We have sensible defaults here, with an optional `alt` prop for consumers to override if they like. 
* We make a required `alt` prop and have no default.

The latter would force consumers to think about what they want, but the former would avoid potential incorrect usages.

Reply:
Maybe the later? Just so it's a little more flexible? I guess that word is a double-edged sword though